lecture4/src/main/java/Helper.java

```diff
@@ -0,0 +1,168 @@
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.http.HttpEntity;
+import org.apache.http.client.entity.UrlEncodedFormEntity;
+import org.apache.http.client.methods.*;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.message.BasicNameValuePair;
+import org.apache.http.util.EntityUtils;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class Helper {
+    private CloseableHttpClient client;
+    private String userName;
+    private String password;
+    private String content;
+    private String secretKey;
+    private ObjectMapper mapper;
+
+
+    public Helper() {
```

Comment:
please create LoginHelper and move register() and login() methods there

Reply:
done